npc/re/instances/MorseCave.txt

```diff
@@ -166,11 +166,6 @@ moro_cav,57,69,3	script	Red Flower#a1	CLEAR_NPC,{
 		mes "to Morocc's lair.";
 		close;
 	}
-	if (isbegin_quest(9318) == 1) {
-		mes "- The Red Flower is closed.";
-		mes "You cannot enter it yet. -";
-		close;
-	}
```

Comment:
These lines prevent the player from re-enter if he has left during the instance. He need to talk to `Senior Tracker` which give the quest delay and complete the quest 9318 (so `isbegin_quest(9318) == 2`).
```
	switch( isbegin_quest(9318) ) {
	case 0:
		//...
	case 1:
		//...
		completequest 9318;// Pursuing Hiding Morroc
		setquest 9319;// Pursuing Hiding Morroc Continues
		close;
```

Reply:
Sorry ,I didn't realize that , thanks.